### PR TITLE
Completed filestack image upload process for profile images 

### PR DESCRIPTION
**Pull request by viviancan**
This change refs #3605 and was the 2nd step after #3594 

This change completes the filestack image upload process for profile images. There were 3 main changes
1. `app/views/profiles/_filestack_profile_image_tag.html.erb` - This snippet is what allows the filestack image to be displayed in the view. The `filestack_image` tag is part of the rails/js filestack gem. 
2. `app/views/profiles/_filestack_profile_image_upload.en.html.erb` This file contains the logic for filestack upload. Once the file is finished uploading, the `onFileUploadFinished` callback function is run that makes the ajax call to update the DB
3. removes the ImageProcessor on the profile_image and also allows for profile_image to be added/updated

### CODE REVIEW

File: app/views/profiles/_filestack_profile_image_upload.en.html.erb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<%= javascript_tag do %>
```
Comment (viviancan) [inline app/views/profiles/_filestack_profile_image_upload.en.html.erb:1]: I tried adding this logic to its own JS file, but I think something weird was happening with the loading/webpacker/etc, where the function was always undefined, so I left it here instead.

File: app/models/account.rb [hunk 2]
```diff
@@ -697,8 +693,26 @@ def reminded_about_survey!
     )
   end
 
+  def default_image_url
```
Comment (viviancan) [inline app/models/account.rb:696]: I'm not sure if this chunk of logic should live somewhere else 🤔 Feedback would be great here!

Comment (shaunxp20) [reply app/models/account.rb:696]: I think this is a good for now, still wrapping my head around everything, a few things I noticed:
- the methods not used outside of `account` should be `private`
- passing `profile_image` to `s3_profile_image_url` and `determine_profile_image_url` shouldn't be necessary

And the thought to just put everything in one method also crossed my mind. 😁 🤔 😅

File: app/views/profiles/_filestack_profile_image_upload.en.html.erb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  function onFileUploadFinished(data) {
+    $("#account_profile_image").val(data.url);
+
+    $.ajax({
```
Comment (shaunxp20) [inline app/views/profiles/_filestack_profile_image_upload.en.html.erb:5]: Def not a blocker, but we could use the Fetch API here.

**Review by shaunxp20 — APPROVED** [review]
LGTM! No blockers (except the existing images not working, but that can be resolved after). Really excited to see this coming together! 👏 💯 🥳 🎈 🙌

File: app/views/profiles/_filestack_profile_image_upload.en.html.erb (hunk 3 above)
Comment (viviancan) [reply app/views/profiles/_filestack_profile_image_upload.en.html.erb:5]: Sounds good, I will look at this process with my update branch :)

File: app/models/account.rb (hunk 2 above)
Comment (viviancan) [reply app/models/account.rb]: I updated this!

**Pull request merged**